rfc-brief.Rmd

```diff
@@ -0,0 +1,225 @@
+But also important is that this document should actually benefit our data contributors.
+<!-- to answer the question of "Why should I glance at this?" -->
+One advantage is allowing our contributors to prioritize submissions accordingly.
+The burden of having to submit data in multiple formats can also be avoided, since preferred formats are clear. 
+These specifications should lead to higher-impact data, leaning towards the most interoperabe formats that make data more likely to be reused (and cited). 
+Finally, this open document lets our community be apprised of the status quo, including specifications currently *not* included, so that they can continue to provide comments.
+
+These current data-sharing specifications have been informed by several sources. 
+First are requirements already established by our funding organizations, which are largely non-negotiable. 
+Second is the feedback received from the community, which was especially influential for data types outside of what is required by default, i.e. more "optional" data types. 
+This feedback was solicited from the research community through an open RFC, from which results are incorporated here. 
+Lastly, journal requirements and other repositories also had some influence on these specifications.
+
+
+## Data types and formats summary
+
+Table \@ref(tab:table-main) summarizes specifications for common data types that are of interest for the NF community.
+These standards should apply for most general cases.
+However, we do note that for some data types, the community comments provided additional considerations that serve to qualify or supplement what is in this table.
+Community results are also summarized in Quantified Results section. 
+
+```{r table-main, fig.label="", layout="l-page", echo=F}
+
+# for gt, use captions instead of fig.label
+table1 <- read.csv("materials/rfc/data_specs.csv")
+table1 %>%
+  select(-c(Description, PrivateNotes)) %>%
+  gt(
+  rowname_col = "Subtype",
+  groupname_col = "ParentType",
+  caption = "Specifications for relevant common data types."
+  ) %>%
+  fmt_markdown(columns = c("Levels")) %>%
+  cols_label(
+    PublicNotes = "Notes"
+  ) %>%
+  # tab_header(title = "Data sharing specifications") %>%
+  tab_footnote(
+    footnote = "Level nomenclature can be cross-referenced with https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables/data-levels, where 'raw' corresponses to Level 1 and 'semi-processed' most closely corresponds to Level 2.",
+    locations = cells_column_labels(columns = Levels)
+  ) %>%
+  tab_style(style = list(cell_text(color = 'gray', style = "italic")), 
+            locations = cells_body(columns = Requirement, 
+                                   rows = Requirement != "required")) %>%
+  tab_options(
+    row_group.background.color = "#FFEFDB",
+    table.font.size = "14px",
+    footnotes.marks = "letters")
+```
+
+
+## Additional considerations
```

Comment:
I think I need a little more context for this section. Maybe a few sentences to preface each of the claims made below?